scripts/meterian-docker

```diff
@@ -30,12 +30,17 @@ else
     DOTNET_LOCAL_DIR=~/.dotnet
     DOTNET_DOCKER_DIR=/home/meterian/.dotnet
 
+    # php specific mappings
+    PHP_LOCAL_DIR=~/.cache
+    PHP_DOCKER_DIR=/home/meterian/.cache
```

Comment:
This is linking all the local caches with the container caches. I would then use different variable names to express that, like CACHES_LOCAL_DIR and CACHES_DOCKER_DIR. I would still leave the comment on top to express that the driver of this change was because of the PHPP integration.